src/__tests__/Search/fetchRevisionByID.test.tsx

```diff
@@ -80,66 +77,74 @@ describe('Search View/fetchRevisionByID', () => {
       />,
     );
 
-    await screen.findAllByRole('button', { name: 'Base' });
-    expect(screen.getAllByText('try')[0]).toBeInTheDocument();
```

Comment:
I thought this wasn't useful in this test (reminder: we want to test things relative to `fetchRevisionByID`).